plugin/sdk_v2/shared/bus/_state.py

```
@@ -0,0 +1,68 @@
+
+
+def ensure_transport(transport: Any | None):
+    return transport if transport is not None else _InMemoryBusTransport()
+
+
+def ensure_state(transport: Any) -> _BusState:
+    state = getattr(transport, "_sdk_v2_bus_state", None)
+    if state is None:
+        state = _BusState()
+        try:
+            setattr(transport, "_sdk_v2_bus_state", state)
```

Ruff (0.15.6)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)